src/loaders.ts

```diff
@@ -16,6 +17,18 @@ const loadJs: LoaderSync = function loadJs(filepath) {
   return result;
 };
 
+const loadJs: LoaderAsync = async function loadJs(filepath) {
+  // This logic is validated by tests running in Node versions that don't
+  // support dynamic imports, like Node 10.
```

Comment:
Is it specific versions of Node 10 that don't support dynamic import syntax?

Using [this site](https://www.jdoodle.com/execute-nodejs-online/), 9.2.0 gives a SyntaxError, but 10.1.0 doesn't.

I think this will fail when the syntax is supported, but the feature is not (f.e. when `--experimental-modules` is needed).

Since v10 EOL is just around the corner, it's probably easier to just wait until then (2021-04-30), and drop support for it.

Reply:
> Since v10 EOL is just around the corner, it's probably easier to just wait until then (2021-04-30), and drop support for it.

I would just drop support for Node.js 10 now. It's 10 days left. This will not be released until then anyway.